app/workers/ons_download_worker.rb

```diff
@@ -0,0 +1,42 @@
+require "open-uri"
+require "zip"
+
+class OnsDownloadWorker < OnsBaseWorker
+  # Example URL: https://www.arcgis.com/sharing/rest/content/items/a2f8c9c5778a452bbf640d98c166657c/data
+  # retrieved by visiting https://geoportal.statistics.gov.uk/search?collection=Dataset&sort=-created&tags=all(PRD_ONSPD)
+  # clicking on the first search result and then copying the link from the download button.
```

Comment:
Think it's worth adding this information to the developer docs if we expect this to be a task that's manually run fairly regularly (not part of this ticket but something to note)

Reply:
Captured as a task in the main trello ticket.